MP Quickstart fails to run with JDK 17

## Environment Details
* Helidon Version: 2.3.0
*  Helidon MP
* JDK version: build 17-ea+24-2164
* OS: MacOS

----------

## Problem Description
The Helidon MP quickstart fails to run using JDK 17:

```
2021.06.02 09:28:50 INFO org.jboss.weld.Event Thread[main,5,main]: WELD-000411: Observer method [BackedAnnotatedMethod] public org.glassfish.jersey.ext.cdi1x.internal.ProcessAllAnnotatedTypes.processAnnotatedType(@Observes ProcessAnnotatedType<?>, BeanManager) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.025 s <<< FAILURE! - in io.helidon.examples.quickstart.mp.MainTest
[ERROR] io.helidon.examples.quickstart.mp.MainTest  Time elapsed: 1.025 s  <<< ERROR!
java.lang.reflect.GenericSignatureFormatError:
Signature Parse error: Expected Field Type Signature
	Remaining input: org.eclipse.microprofile.openapi.annotations.enums.SchemaType
	at io.helidon.examples.quickstart.mp.MainTest.startTheServer(MainTest.java:27)
```

## Steps to reproduce
1. Install JDK 17 preview and make sure you are using it: `java --version`
2. Generate a project using the 2.3.0 Helidon MP quickstart
3.  `mvn clean install`

--- a/dependencies/pom.xml
+++ b/dependencies/pom.xml
@@ -72,7 +72,7 @@
         <version.lib.jaxb-core>2.3.0.1</version.lib.jaxb-core>
         <version.lib.jaxb-impl>2.3.2</version.lib.jaxb-impl>
         <version.lib.jaxrs-api>2.1.6</version.lib.jaxrs-api>
-        <version.lib.jboss.classfilewriter>1.2.4.Final</version.lib.jboss.classfilewriter>
+        <version.lib.jboss.classfilewriter>1.2.5.Final</version.lib.jboss.classfilewriter>
         <version.lib.jboss.logging>3.2.1.Final</version.lib.jboss.logging>
         <version.lib.jboss.transaction-api>1.0.0.Final</version.lib.jboss.transaction-api>
         <version.lib.jboss.transaction-spi>7.6.0.Final</version.lib.jboss.transaction-spi>

